### PR TITLE
KOGITO-2431: Fixed snapshots related to DE

### PR DESCRIPTION
**Pull request by Sara4994**
Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

File: ui-packages/packages/common/src/components/Molecules/DomainExplorerTable/tests/DomainExplorerTable.test.tsx [hunk 1]
```diff
@@ -15,7 +16,7 @@ describe('Domain Explorer Table Component', () => {
     mockUseEffect();
     mockUseEffect();
   });
-  it('Snapshot test', () => {
+  it('Snapshot test with default props', async () => {
```
Comment (jstastny-cz) [inline ui-packages/packages/common/src/components/Molecules/DomainExplorerTable/tests/DomainExplorerTable.test.tsx:21]: I would maybe keep this test as the only one that actually uses the patternfly table component and mock in the rest.

The goal is to make the snapshot shorter - this way we make sure that the component renders fine with default props and then in other snapshots just focus on the behavior (often it is emtpy-state vs. table rednered - but we don't need to see the contents of the table, it is enough to see that table component is being used)

File: ui-packages/packages/common/src/components/Molecules/DomainExplorerTable/tests/DomainExplorerTable.test.tsx [hunk 2]
```diff
@@ -102,20 +103,24 @@ describe('Domain Explorer Table Component', () => {
...
+        <DomainExplorerTable {...props} />
+      </MockedProvider>,
+      'DomainExplorerTable'
+    );
     wrapper.update();
-    wrapper.setProps({});
     expect(wrapper).toMatchSnapshot();
   });
-  it('Boolean assertions-false', () => {
+  it('Boolean assertions-false', async () => {
```
Comment (jstastny-cz) [inline ui-packages/packages/common/src/components/Molecules/DomainExplorerTable/tests/DomainExplorerTable.test.tsx]: here I am still not clear on the description of the test, one is `Boolean assertions-false`, second `Boolean assertions-true` - what is the difference?

Comment (jstastny-cz) [reply ui-packages/packages/common/src/components/Molecules/DomainExplorerTable/tests/DomainExplorerTable.test.tsx]: Hum, the snapshots for the two tests are identical, just differ based on passed values (state `COMPLETED` vs. `PENDING` in one process instance). So unless there's some reason I am missing I would remove both the snapshots, cause they're identical also to `Snapshot test with default props`. If this is due to coverage - we might need to test differently - cause there's nothing interesting in the resulting snapshot.

File: ui-packages/packages/common/src/components/Molecules/DomainExplorerTable/tests/DomainExplorerTable.test.tsx [hunk 3]
```diff
@@ -404,19 +417,24 @@ describe('Domain Explorer Table Component', () => {
...
+      <MockedProvider>
+        <DomainExplorerTable {...props} />
+      </MockedProvider>,
+      'DomainExplorerTable'
+    );
     wrapper.update();
     expect(wrapper).toMatchSnapshot();
   });
-  it('check false value of isLoadingMore', () => {
+  it('check false value of isLoadingMore', async () => {
```
Comment (jstastny-cz) [inline ui-packages/packages/common/src/components/Molecules/DomainExplorerTable/tests/DomainExplorerTable.test.tsx:366]: is empty-state in snapshot here expected?

File: ui-packages/packages/common/src/components/Molecules/DomainExplorerTable/tests/DomainExplorerTable.test.tsx [hunk 4]
```diff
@@ -305,19 +313,24 @@ describe('Domain Explorer Table Component', () => {
...
+      <MockedProvider>
+        <DomainExplorerTable {...props} />
+      </MockedProvider>,
+      'DomainExplorerTable'
+    );
     wrapper.update();
     expect(wrapper).toMatchSnapshot();
   });
-  it('check zero offset', () => {
+  it('check zero offset', async () => {
```
Comment (jstastny-cz) [inline ui-packages/packages/common/src/components/Molecules/DomainExplorerTable/tests/DomainExplorerTable.test.tsx:261]: Should zero offset result in empty-state? Or needs the test to be fixed?

File: ui-packages/packages/common/src/components/Organisms/DomainExplorer/tests/__snapshots__/DomainExplorer.test.tsx.snap [hunk 5]
```diff
@@ -1505,7 +873,1313 @@ exports[`Domain Explorer Container component Snapshot test 1`] = `
...
+            ],
+          },
+        ]
+      }
+      rows={Array []}
+      selected={Array []}
+      setRows={[Function]}
+      tableLoading={false}
+    />
+    <LoadMore
```
Comment (jstastny-cz) [inline ui-packages/packages/common/src/components/Organisms/DomainExplorer/tests/__snapshots__/DomainExplorer.test.tsx.snap]: can we try mocking LoadMore too?

File: ui-packages/packages/common/src/components/Organisms/DomainExplorer/tests/__snapshots__/DomainExplorer.test.tsx.snap [hunk 6]
```diff
@@ -1505,7 +873,1313 @@ exports[`Domain Explorer Container component Snapshot test 1`] = `
...
+      Object {
+        "flight": Array [
+          "departure",
+        ],
+      },
+    ]
+  }
+  rememberedSelections={Array []}
+>
+  <DataToolbar
```
Comment (jstastny-cz) [inline ui-packages/packages/common/src/components/Organisms/DomainExplorer/tests/__snapshots__/DomainExplorer.test.tsx.snap:956]: Can we mock DataToolbar now?

File: ui-packages/packages/management-console/src/components/Templates/DomainExplorerLandingPage/tests/__snapshots__/DomainExplorerLandingPage.test.tsx.snap [hunk 7]
```diff
@@ -1,242 +1,164 @@
...
+                          Home
+                        </a>
+                      </LinkAnchor>
+                    </Link>
+                    <span
+                      className="pf-c-breadcrumb__item-divider"
                     >
-                      <li
-                        className="pf-c-breadcrumb__item"
+                      <AngleRightIcon
```
Comment (jstastny-cz) [inline ui-packages/packages/management-console/src/components/Templates/DomainExplorerLandingPage/tests/__snapshots__/DomainExplorerLandingPage.test.tsx.snap]: I think we can mock all the icons used here, it'll save some space in snapshots.

File: ui-packages/packages/management-console/src/components/Templates/DomainExplorerPage/tests/__snapshots__/DomainExplorerPage.test.tsx.snap [hunk 8]
```diff
@@ -98,602 +98,252 @@ exports[`Domain Explorer Dashboard component Check error response for getQueryFi
...
+                        </a>
+                      </LinkAnchor>
+                    </Link>
+                    <span
+                      className="pf-c-breadcrumb__item-divider"
                     >
-                      <div
-                        className="pf-c-data-toolbar__expandable-content"
-                        id="data-toolbar-with-chip-groups-expandable-content-4"
+                      <AngleRightIcon
```
Comment (jstastny-cz) [inline ui-packages/packages/management-console/src/components/Templates/DomainExplorerPage/tests/__snapshots__/DomainExplorerPage.test.tsx.snap]: could we mock the icons to shorten the snapshot?

**Review by jstastny-cz — CHANGES_REQUESTED** [review]

File: ui-packages/packages/common/src/components/Molecules/DomainExplorerTable/tests/DomainExplorerTable.test.tsx (hunk 3 above)
Comment (Sara4994) [reply ui-packages/packages/common/src/components/Molecules/DomainExplorerTable/tests/DomainExplorerTable.test.tsx:366]: @jstastny-cz yes it is, as per the condition provided.

File: ui-packages/packages/common/src/components/Organisms/DomainExplorer/tests/__snapshots__/DomainExplorer.test.tsx.snap (hunk 5 above)
Comment (Sara4994) [reply ui-packages/packages/common/src/components/Organisms/DomainExplorer/tests/__snapshots__/DomainExplorer.test.tsx.snap]: this is done

File: ui-packages/packages/common/src/components/Molecules/DomainExplorerTable/tests/DomainExplorerTable.test.tsx (hunk 2 above)
Comment (Sara4994) [reply ui-packages/packages/common/src/components/Molecules/DomainExplorerTable/tests/DomainExplorerTable.test.tsx]: This is made as one single snapshot tests all state values

File: ui-packages/packages/common/src/components/Molecules/DomainExplorerTable/tests/DomainExplorerTable.test.tsx (hunk 4 above)
Comment (Sara4994) [reply ui-packages/packages/common/src/components/Molecules/DomainExplorerTable/tests/DomainExplorerTable.test.tsx:261]: This is fixed, now it renders table not empty-state

File: ui-packages/packages/common/src/components/Organisms/DomainExplorer/tests/__snapshots__/DomainExplorer.test.tsx.snap (hunk 6 above)
Comment (Sara4994) [reply ui-packages/packages/common/src/components/Organisms/DomainExplorer/tests/__snapshots__/DomainExplorer.test.tsx.snap:956]: This is done

File: ui-packages/packages/management-console/src/components/Templates/DomainExplorerLandingPage/tests/__snapshots__/DomainExplorerLandingPage.test.tsx.snap (hunk 7 above)
Comment (Sara4994) [reply ui-packages/packages/management-console/src/components/Templates/DomainExplorerLandingPage/tests/__snapshots__/DomainExplorerLandingPage.test.tsx.snap]: Mocked Breadcrumb as suggested

File: ui-packages/packages/management-console/src/components/Templates/DomainExplorerPage/tests/__snapshots__/DomainExplorerPage.test.tsx.snap (hunk 8 above)
Comment (Sara4994) [reply ui-packages/packages/management-console/src/components/Templates/DomainExplorerPage/tests/__snapshots__/DomainExplorerPage.test.tsx.snap]: Mocked Breadcrumb as suggested

File: ui-packages/packages/common/src/components/Organisms/DomainExplorer/tests/DomainExplorer.test.tsx [hunk 9]
```diff
@@ -1,15 +1,28 @@
...
 import useGetColumnPickerAttributesQuery = GraphQL.useGetColumnPickerAttributesQuery;
 jest.mock('react-apollo');
 
+jest.mock(
+  '../../../Molecules/DomainExplorerColumnPicker/DomainExplorerColumnPicker'
+);
+jest.mock('../../../Molecules/DomainExplorerTable/DomainExplorerTable');
+jest.mock('../../../Atoms/LoadMore/LoadMore');
+
+const MockedDatToolbar = (): React.ReactElement => {
```
Comment (jstastny-cz) [inline ui-packages/packages/common/src/components/Organisms/DomainExplorer/tests/DomainExplorer.test.tsx]: typo MockedDatToolbar -> MockedDataToolbar

**Review by jstastny-cz — CHANGES_REQUESTED** [review]
Just a typo in component name, otherwise looks great.

File: ui-packages/packages/common/src/components/Organisms/DomainExplorer/tests/DomainExplorer.test.tsx (hunk 9 above)
Comment (Sara4994) [reply ui-packages/packages/common/src/components/Organisms/DomainExplorer/tests/DomainExplorer.test.tsx]: Corrected :+1:

**Review by jstastny-cz — APPROVED** [review]

**Review by nmirasch — APPROVED** [review]

**Review by AjayJagan — APPROVED** [review]

**Review by cristianonicolai — APPROVED** [review]

**Pull request merged**